index.html

```diff
@@ -5,126 +5,111 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>All about trees Group6</title>
-        <!-- this is the link to the css file -->
-
     <link href="./styles/index.css" rel="stylesheet" />
-    
   </head>
-  <header>
-    <nav>
-      <div class="logo">
-        <img src="./public/logo-group-6.jpg" alt="all about trees" />
+
+  <body>
+    <header>
+      <nav>
+        <div class="logo">
+          <img src="./public/logo-group-6.jpg" alt="all about trees" />
+        </div>
+        <ul>
+          <li>Home</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+      </nav>
+      <div class="title">
+        <h1>All About tree</h1>
+        <p>Save the tree save the life</p>
       </div>
-      <ul>
-        <li><a href="#"></a>Home</li>
-        <li><a href="#"></a>About</li>
-        <li><a href="#"></a>Contact</li>
-      </ul>
-    </nav>
-    <div class="title">
-      <h1>All About tree</h1>
-      <p>Save the tree save the life</p>
-    </div>
-    <div class="intro">
-      <p>
-        Trees are an important part of every community. Since the beginning,
-        trees have furnished us with two of life’s essentials, food and oxygen.
-        Trees control climate by moderating the effects of the sun, rain and
-        wind. Leaves absorb and filter the sun’s radiant energy, keeping things
-        cool in summer. Trees also preserve warmth by providing a screen from
-        harsh wind. In addition to influencing wind speed and direction, they
-        shield us from the downfall of rain, sleet and hail. Trees also lower
-        the air temperature and reduce the heat intensity of the greenhouse
-        effect by maintaining low levels of carbon dioxide.
-      </p>
-    </div>
-    <button class="btn-video">Video</button>
-  </header>
-  <!-- Main body area-->
-  <body class="body">
-    <br />
+      <div class="intro">
```

Comment:
this part doesn't need to be  in header, we can use header section for nav bar and title